reference/scalability_guide/more_than_100k_products_to_export.rst

```diff
@@ -1,16 +1,16 @@
 More than 100k products to export?
 ----------------------------------
 
-We had use cases where customers would export 270k products and had issues with the memory usage.
+In the past, we encountered use cases where partners would export 270k products and experienced issues with the memory usage.
 
-Most of the massive operations of the PIM like imports and exports are bulked by page of lines (or objects) to avoid too large memory usage.
+Most of PIM's massive operations like imports and exports are #bulked by page of lines (or objects)# to avoid too large memory usage. #bulked by page of lines ?""
```

Comment:
So maybe we can say:
"Most of PIM's massive operations like imports and exports processes the products iteratively via a size configured subsets of products in order to minimize the memory usage"